Nested Drawer (#25)
pxblue/react-component-library: docs/InfoListItem.md

| leftComponent     | Component to render on the left side    | `React.Component`                                  | no       |                     | `<ListItemText/>`                       |
| onClick           | A function to execute when clicked      | `function`                                         | no       |                     | `() => console.log('pressed')`          |
| rightComponent    | Component to render on the right side   | `React.Component`                                  | no       |                     | `<ChannelValue/>`                       |

> | ripple            | Whether we want any ripple effect       | `boolean`                                          | no       | false               |                                         |

Reviewer: ```suggestion
| ripple            | Whether to apply material ripple effect on click       | `boolean`                                          | no       | false               |                                         |
```
